manual/custom_codecs/README.md

```diff
@@ -296,12 +296,12 @@ public class AddressCodec extends TypeCodec<Address> {
 
     @Override
     public Address parse(String value) throws InvalidTypeException {
-        return value == null || value.isEmpty() || value.equals(NULL) ? null : toAddress(innerCodec.parse(value));
+        return value == null || value.isEmpty() || value.equalsIgnoreCase("NULL") ? null : toAddress(innerCodec.parse(value));
```

Comment:
Nit: could you split that into 3 lines (before `?` and `:`)?
With the default Github markdown rendering, the code overflows on the right.